main.tf

```diff
@@ -103,6 +103,15 @@ resource "ibm_scc_instance_settings" "scc_instance_settings" {
   }
 }
 
+resource "ibm_iam_authorization_policy" "en_s2s_policy" {
+  count                       = var.skip_en_s2s_auth_policy || var.existing_scc_instance_crn != null ? 0 : 1
```

Comment:
We might need to test if an SCC resource block is used in the same terraform as this module, and we use existing instance variable, will that cause terraform plan to error with unknown variable value in count.

Reply:
it would fail, but its unlikely a user will have existing SCC in same state when running this module (otherwise they would just use this module in to create new instance). We use `existing_scc_instance_crn` in other counts in this module already, and it hasnt been an issue for anyone yet